### DISCUSSION TITLE
achievemnets

### DISCUSSION THREAD

**Discussion by aleebaba3594** [community/community#35034] [Other Feature Feedback, Questions, & Ideas] [answered]
How do I get the Galaxy Brain badge?

**Comment by radar-blip** [comment, marked as answer]
Here is a list of all achievement badges and how to get them: [Achievement Badges](https://github.com/Schweinepriester/github-profile-achievements)

**Reply by aleebaba3594** [reply]
can you open a discussion so that i'll able to comment....want to get the badge 
thanks in advance

**Comment by github-staff** [comment]
Because of an influx of users creating low quality, nonsensical or spam posts in the Community with the aim of earning certain Achievements, we’ve made the difficult decision to disable the ability to earn Achievements in our Community.

Note that this change does not affect your ability to earn badges in other projects and communities across GitHub. This change only applies to GitHub’s Community Discussions.

You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Let us know if you have any other questions 🚀

**Comment by Pavel-Shcherbo** [comment]
How do I get a Galaxy Brain badge if I have answered 20 discussions?
